src/ORM/Table.php

```diff
@@ -1226,6 +1208,35 @@ public function get($primaryKey, $options = [])
     }
 
     /**
+     * @param int|string|array $primaryKey Primary key value.
+     *
+     * @return array Conditions.
+     */
+    public function primaryKeyToConditions($primaryKey)
```

Comment:
Lots of methods _could_ be useful. But each public method we have creates more contract surface with our users. Until we have a need to expose 'helper' methods I would leave them protected. We can always increase visibility later, but we can hide methods as easily.